### ISSUE TITLE
build broken on Windows

### ISSUE THREAD

**Issue by BurtHarris** [preactjs-templates/default#36]
The devDependency `per-env` has had a known Windows bug that hasn't been resolved for years.    

An alternative package, [`by-node-env`](https://www.npmjs.com/package/by-node-env) which seems better maintained, may offer a drop-in replacement.

**Comment by ForsakenHarmony** [comment]
is there an alternative that's 0 dependencies like the old one?

**Comment by BurtHarris** [comment]
@ForsakenHarmony , you say "0 dependencies like the old one", but I can't find anywhere in the git history where that was true.    It looks like the initial check-in depended on **if-env**, which was later switch to a dependency on **per-env**.  

But I think I agree in principal with your point.    After all, I was just following the instruction on the [getting started ](https://preactjs.com/cli/getting-started) page and ran into a failure at the step which instructed me to type `npm start`.     

Perhaps that getting started instruction should read `preact watch` instead! 
Then we could tear out the start scripts and related dependencies...

**Comment by ForsakenHarmony** [comment]
I meant https://www.npmjs.com/package/per-env having 0 dependencies to other packages

**Comment by BurtHarris** [comment]
https://www.npmjs.com/package/cross-per-env seems have been a fork trying to fix the issue but the author didn't even update the README to reflect the name change.   Also https://www.npmjs.com/package/@gendall/per-env is another fork, but there the docs look non-existant.

If no `start` script is specified in **package.json**, then the [npm start ](https://docs.npmjs.com/cli/start.html)command is documented to run a `server.js` file instead, that could be leveraged.   But my read is that this was feature several people thought was cool, but nobody bothered to maintain.   Its likely to always be abandon-ware unless it gets incorporated into something more substantial like **npm** itself.

I'd suggest the smart move would be to cut the wiz-bang feature and document developer launch and production launch as two different life-cycle scenarios calling for different commands

**Comment by benhormann** [comment]
I'd also like to point out that the current dependencies/devDependencies split doesn't make sense... If you do `export NODE_ENV=production && npm install && npm start` it won't run.
It should look more like this:
```
  "dependencies": {
    "per-env": "^1.0.2",
    "sirv-cli": "^0.4.5"
  },
  "devDependencies": {
    ...
    // build dependencies
    // test dependencies
    ...
  },
```
Except 'preact build' is in the start:production script, so currently all build dependencies should be under "dependencies".

### TL;DR
As it stands preact-cli is required to run, so why not add `preact start` to it?

**Comment by ForsakenHarmony** [comment]
I don't really like the current behaviour, don't think it adds much if anything

**Comment by ForsakenHarmony** [comment]
We've decided to get rid of `per-env`
